Containerfile

```diff
@@ -38,6 +38,7 @@ COPY dist2src /src/dist2src
 COPY .git /src/.git
 
 WORKDIR /src
+RUN pip3 install git+https://github.com/packit/requre.git
```

Comment:
please add requre to this list: https://github.com/packit/dist-git-to-source-git/blob/master/files/install-deps.yml#L58

Reply:
Thanks, that was just an example, how it may work. so it is not expected to be merged as it is.
In case It will does sense, then we can merge it and use, actually with haproxy tar archive is 2.6M big, it is not so much but for kernel or similar it may be much bigger, could solve situatim when testing env does not have network access, It could decrease network transfers, but maybe it is still too big to be part of tests.  and it is not comprehensive, because I think that also `CliRunner` calls it as external program if I understand it well, so  it is not recorded by requre.

Reply:
oki, I agree that we are sacrificing storage in git repo in favour of speed (= not clone the repo, create it locally)